tekton/utils/pipeline.go

```diff
@@ -139,7 +145,32 @@ func (pr *PipelineRef) ToTektonPipelineRef() *tektonv1.PipelineRef {
 	return tektonPipelineRef
 }
 
-// GetTektonParams returns the ParameterizedPipeline []Param as []tektonv1.Param.
+// GetTektonParams returns the PipelineRef's configured parameters as []tektonv1.Param.
+// Currently this only includes ociStorage if specified.
+func (pr *PipelineRef) GetTektonParams() []tektonv1.Param {
```

Comment:
I'm a little confused by how the changes in this file work. I thought it'd be something a little more like https://github.com/konflux-ci/release-service/blob/main/tekton/utils/pipeline_run_builder.go#L220
This isn't the nicest, but we don't plan to support a growing number of parameters anyways